decapodes/python/model.py

```diff
@@ -69,25 +138,25 @@ class AbstractNamedDecapode(AbstractDecapode):
     pass
 
 
-class SymDecapode(AbstractDecapode):
+class DecapodeSym(AbstractDecapode):
```

Comment:
it looks like we flipped the class name components here. 

Responsible line is here, right?
https://github.com/DARPA-ASKEM/Model-Representations/blob/46d0f17b87966cdd0153343d7ec9e716f73ff4e5/decapodes/intertypes/model.it#L72

Reply:
It looks like it's flipped in a few places. I haven't touched the model intertypes so this might be more of an @olynch question

Reply:
It is `SymSummationDecapode` on upstream master. And I don't see any upstream PRs to change it. 

https://github.com/AlgebraicJulia/DiagrammaticEquations.jl/blob/main/src/decapodeacset.it

I think we should use the one from upstream, which has the benefit of less churn on this repo too.